brian2/parsing/bast.py

```diff
@@ -204,6 +207,7 @@ def render_BoolOp(self, node):
         node.dtype = 'boolean'
         for subnode in node.values:
             if subnode.dtype!='boolean':
+                print subnode
```

Comment:
Debugging with print statements, who would do such a thing ;) Thanks for spotting this.